src/Components/Home.js

```diff
@@ -1,19 +1,47 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
 import HomeMenuPosition from "./HomeMenuPosition";
 import UserNavBar from "./UserNavBar";
+import ChooseSyllabary from "./ChooseSyllabary";
 
 class Home extends Component {
+  state = {
+    isShown: false
+  };
+
+  //displaying choose syllabary component
+  handleToggleClick = () => {
+    console.log("click!");
+    this.setState({
+      isShown: !this.state.isShown
+    });
+  };
+
   render() {
+    const { isShown } = this.state;
+    const style = {
+      backgroundColor: "#c0ffee",
+      display: isShown ? "block" : "none"
+    };
     return (
       <section className="home">
+        <ChooseSyllabary style={style} />
```

Comment:
Instead of using `display` to show/hide this element try this: 

```jsx
<section className="home">
  {isShown && <ChooseSyllabary />}

```

Also take the `backgroundColor` and apply it inside the `ChooseSyllabary` component. No need for doing it here :)